src/ir/names.h

```diff
@@ -27,8 +27,8 @@ namespace Names {
 // modify existing names
 inline void ensureNames(Function* func) {
   std::unordered_set<Name> seen;
-  for (auto& pair : func->localNames) {
-    seen.insert(pair.second);
+  for ([[maybe_unused]] auto& [_, name] : func->localNames) {
```

Comment:
Will we get errors without the `maybe_unused` attribute? I ask because it's kind of ugly in the source 😕

Reply:
Yes, I got unused variable warning. I also think it will be great avoid such attribute. May be add `_` to "enablelist" or something like this, but I don't know if this possible